components/uniset/src/uniset.rs

```diff
@@ -13,7 +18,8 @@ const BMP_MAX: u32 = 0xFFFF;
 ///
 /// Provides exposure to membership functions and constructors from serialized [`UnicodeSets`](UnicodeSet)
 /// and predefined ranges.
-#[derive(Debug, PartialEq, Hash, Eq)]
+#[derive(Debug, PartialEq, Hash, Eq, Clone)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
I thought you were going to remove the Serde auto-impl.  It's still here.  Please fix.

To be clear, a better outcome would be for you to actually implement the manual Serde impl, since we will need it next week to unblock Iain's intern.

Reply:
Sorry, I didn't see your `"deserialize_inv_list"` function.